src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
+@import"variables";
+
+html {
+  font-family: "Arial", "Helvetica", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 35px auto;
+  width: $calendarWidth;
+  padding: $calendarPadd;
+  gap: $dayGap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $daySize;
+    width: $daySize;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: color 0.5s, transform 0.5s;
+    font-size: 30px;
+  }
+
+  &__day:hover {
+    background-color: #ffc0cb;
+    cursor: pointer;
```

Comment:
Cursor property should be set in the base class not in hover state. This would make it clear that the element is interactive even before hovering.